### PR TITLE
Update required tool version to 15.3

### PR DESCRIPTION
**Pull request by blairconrad**
Supports #1185.

I thought of this just after merging #1187.

### CODE REVIEW

File: how_to_build.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ At the time of writing the build is only confirmed to work on Windows using the
 
 1. Ensure you have .NET framework 4.6.1 or later installed.
 
-1. Ensure you have Visual Studio 2017 or MSBuild 15 installed.
+1. Ensure you have Visual Studio 2017 or MSBuild 15.3 installed.
```
Comment (thomaslevesque) [inline how_to_build.md]: 15.3 is the version of Visual Studio 2017, not MSBuild

Comment (thomaslevesque) [inline how_to_build.md]: OK, apparently it's also the version of MSBuild... I still find the wording confusing, though. I think it's supposed to mean "(Visual Studio 2017 or MSBuild) 15.3", but I read it as "(Visual Studio 2017) or (MSBuild 15.3)"

Comment (blairconrad) [inline how_to_build.md]: Understood.

Comment (blairconrad) [inline how_to_build.md]: Better?

Comment (thomaslevesque) [inline how_to_build.md]: Yes 👍

**Comment by thomaslevesque** [comment]
Thanks @blairconrad !

**Comment by blairconrad** [comment]
Thank you.

**Pull request merged**